vignettes/funbiogeo.Rmd

```diff
@@ -20,405 +21,296 @@ knitr::opts_chunk$set(
 
-The package `funbiogeo` comes with the example dataset `site_species`.
+Note that here we are talking about sites in an abstract way.
+These can be plots, assemblages, of whatever set of species
+youre interested in.
 
-```{r 'load-sites-species'}
-## Load site x species example dataset ----
-data("site_species", package = "funbiogeo")
-```
+The package `funbiogeo` comes with the example dataset `site_species`.
+Let's look at it:
 
-```{r 'preview-sites-species', echo = FALSE}
+```{r preview-sites-species, echo = FALSE}
 knitr::kable(head(site_species[ , 1:4], 10), 
              caption   = "Format of site x species dataset", 
              align     = c("c", "c", "c", "c"))
 ```
 
-```{r 'print-sites-ids-2'}
-## First column contains sites names ----
-site_species[1:4, 1]
-```
-
-```{r 'print-species-ids-1'}
-## Column names after the first one contain species names ----
-colnames(site_species)[2:4]
-```
-
-To create your own site x species dataset you can use the function 
-`fb_format_site_species()`. See below the section ***Long format***.
-
-
-
-### Species x Traits
-
-This dataset contains traits values for studied species. It can be either a 
-`matrix` or a `data.frame` and row names must contain species names. Columns 
-must contain only traits variables and species must be unique. No additional 
-columns are accepted.
-
-Note that traits values cannot vary across sites for a given species.
+The example dataset contains the occurrence of the 149 above-mentioned species
+across 1505 sites of 0.5° by 0.5° resolution.
 
-The package `funbiogeo` comes with the example dataset `species_traits`.
 
-```{r 'load-species-traits'}
-## Load species x traits example dataset ----
-data("species_traits", package = "funbiogeo")
-```
+### Site x Locations
 
-```{r 'preview-species-traits', echo = FALSE}
-knitr::kable(head(species_traits, 4), 
-             caption   = "Format of species x traits dataset", 
-             align     = c("r", "r", "r", "r", "r", "r"))
-```
+This object contains the geographical location of the sites.
+It should be an `sf` object
+from the [`sf` package](https://cran.r-project.org/package=sf). The sites can
+have arbitrary shapes: points, regular polygons, irregular polygons, or even
+line transects! To make sure that your data is well plotted you should specify
+the Coordinate Reference System (CRS) of this object.
 
-```{r 'print-species-ids-2'}
-## First column contains species names ----
-species_traits[1:4, 1]
-```
+The package `funbiogeo` comes with the example dataset `site_locations` defining
+the location of the 1505 pixels of 0.5° by 0.5° resolution.
 
-```{r 'print-traits-ids-1'}
-## Column names (apart from the first) contain traits names ----
-colnames(species_traits)[-1]
+```{r preview-sites-locs, echo = FALSE}
+knitr::kable(head(site_locations, 10), 
+             caption   = "Format of site x locations dataset", 
+             align     = c("r", "r"))
 ```
```

Comment:
Maybe do not use `kable()` but just print the object to see the spatial aspect (extent, CRS, etc.)?